File: 01_intro/problems/03_array_static/array_static.f90
program array_static
  integer, parameter :: n = 10
  real(8) :: a(n)
  real(8) :: s
  integer :: i
  ! 配列を a(i) = i の二乗 で埋める (添字は 1..n)
  do i = 1, n
     a(i) = real(i, 8) * real(i, 8)
  end do
  ! 合計を求める
  s = 0.0d0
  ! TODO: 配列 a の全要素を s に足し込むループを書け.
  print "(a,i0,a,f0.0)", "sum of squares 1..", n, " = ", s
end program array_static

File: 01_CF_primer/problems/03_array_static/array_static.cpp
#include <cstdio>

int main() {
  const int n = 10;
  double a[n];
  /* 配列を a[i] = (i+1) の二乗 で埋める */
  for (int i = 0; i < n; i++) {
    a[i] = (i + 1) * (i + 1);
  }
  /* 合計を求める */
  double s = 0.0;
  // TODO: 配列 a の全要素を s に足し込むループを書け.
  printf("sum of squares 1..%d = %.0f\n", n, s);
  return 0;
}
